lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -2760,16 +2835,25 @@ static llvm::Error checkAllNodesSupported(const Function &F, const Backend &B) {
 /// profiling, and FP16 conversion.
 static void transformForPrecisionMode(const Backend &B, Function *F,
                                       CompilationContext &cctx) {
+  // Log scope info.
+  LOG_SCOPE(F->getLogContext(), "transformForPrecisionMode")
   const PrecisionConfiguration &precConfig = cctx.precisionConfig;
 
   switch (precConfig.quantMode) {
   case QuantizationMode::Profile: {
     assert(cctx.bindings);
+
+    // Log scope info.
+    LOG_SCOPE(F->getLogContext(), "glow::profileQuantization")
```

Comment:
Why did you decide to log scopes for each case here inside `transformForPrecisionMode()`, instead of logging them inside the function call itself (e.g. inside `profileQuantization()` below?

Reply:
quantization::quantizeFunction calls FunctionConverter::convert(), which is same as for precConfig.convertToFP16.

So I moved the LOG_SCOPE outside to differentiate them